Dfe.Data.SearchPrototype/Infrastructure/Tests/Integration/TestHarness/MyServiceProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Dfe.Data.Common.Infrastructure.CognitiveSearch.Filtering;
+using Dfe.Data.Common.Infrastructure.CognitiveSearch.SearchByKeyword;
+using Dfe.Data.SearchPrototype.Infrastructure.Tests.TestDoubles;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Dfe.Data.SearchPrototype.Infrastructure.Tests.Integration.TestHarness;
+
+public class MyServiceProvider
+{
+    public IServiceProvider SetUpServiceProvider(IConfiguration config)
```

Comment:
Just wondering if it would be useful to pass a delegate to the renamed 'MyServiceProvider' so we can use this class across multiple composition root tests?

Reply:
Ohh, and probs needs renaming :)

Reply:
I've renamed. 
On the other issue I would prefer to think about that if and when the situation of re-using this class happens

Reply:
Yeah that's fair enough. It was just putting some thoughts down but it kinda leans into some of the things we wanna cover on testing strategies etc, maybe the need for some shared test utilities